sympy/stats/joint_rv.py

```diff
@@ -30,7 +32,7 @@ def __new__(cls, sym, dist):
 
     @property
     def set(self):
-        return self.domain.set
+        return self.distribution.set
```

Comment:
why this?

Reply:
I think self.domain should return an object of `RandomDomain`, which is not yet implemented for `JointPSpace`.